tests/queries/0_stateless/00020_optimizer.sql

```diff
@@ -8,3 +8,6 @@ SELECT '';
 
 SELECT 'group by push down: push alias to group by';
 EXPLAIN select (number+1) as c1, (number%3+1) as c2 from numbers_mt(10000) group by c2;
+
+SELEE 'projection push down: push (max_threads) to readdatasource'
+EXPLAIN select max_thread from system.settings where max_block_size > 10 group by default_db;
```

Comment:
00020_optimizer.reference is the *.sql execute result.
To get the result:
1. make run -- start a fuse-query server for fuse-test
2. cd tests
3. ./fuse-test